python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -256,6 +256,7 @@ def setDefaults(self):
         self.photoCal.match.sourceSelection.flags.good = []
         self.photoCal.match.sourceSelection.flags.bad = ['flag_badStar']
         self.photoCal.match.sourceSelection.doUnresolved = False
+        self.photoCal.match.sourceSelection.doRequirePrimary = False
```

Comment:
Why `False`?  Is the `detect_isPrimary` column not in your catalog (should it be?!)?

Reply:
In this particular case, this {{photoCal}} is all about a post-processing per-tract re-calibration of the already calibrated stars, and is only used in tests to ensure that things match up.  So there's nothing about isPrimary here.